MultiplayerMod/Core/Unity/Dev/DevToolSceneInspectorPatch.cs

```diff
@@ -0,0 +1,21 @@
+﻿using HarmonyLib;
+using ImGuiNET;
+
+namespace MultiplayerMod.Core.Unity.Dev;
+
+[HarmonyPatch(typeof(DevToolSceneInspector))]
+public class DevToolSceneInspectorPatch {
+
+    [HarmonyPrefix]
+    [HarmonyPatch(nameof(DevToolSceneInspector.DisplayField))]
+    private static bool DisplayFieldPrefix(ref bool __result, string name, System.Type ft, ref object obj) {
```

Comment:
Could you add a comment what is it? And why is it in core package?

Reply:
It's for the built-in dev tools (the ImGui menu).
And this is an additional type mapping for the object inspector so we can see all the values in-game.
Maybe it's better be placed in `Game` okay.

Reply:
<img width="304" alt="image" src="https://github.com/zuev93/oni_multiplayer/assets/10905879/fb30ca4c-dbed-421a-a75d-fd15c219032b">

Reply:
<img width="295" alt="image" src="https://github.com/zuev93/oni_multiplayer/assets/10905879/02b26744-7738-498f-9103-f7cd922a7265">
As you can see it doesn't have support for 64 bit types.🤷‍♂️